api/services/ocean/spark/schemas/oceanSparkIssues.yaml

```diff
@@ -0,0 +1,41 @@
+  description: An error that happened during a Spark application
+  type: object
+  properties:
+    internalId:
```

Comment:
do we need to expose internalId, same question for appInternalId? I know Manu is removing it from other API endpoints

Reply:
You're right, they are not yet removed but will be. I'm removing them from the documentation.